### ISSUE TITLE
[Bug] get_account_state result differs from spec

### ISSUE THREAD

**Issue by yanivmo** [libra/libra#4160]
# 🐛 Bug

Calling `get_account_state` returns an object with a property called "balances", while in [the protocol spec](https://github.com/libra/libra/blob/testnet/json-rpc/json-rpc-spec.md#account---type) it is specified as "balance".

## To reproduce

Run this command:
```shell
curl -X POST --data '{"jsonrpc":"2.0","method":"get_account_state","params":["f16df8bc9bfbba501630912814d56e5f"],"id":1}' --header 'content-type: application/json' https://client.testnet.libra.org | jq .
```

This is the response I get:
```json
{
  "id": 1,
  "jsonrpc": "2.0",
  "result": {
    "authentication_key": "9361a14097c8e9b7e284ec600ea45871f16df8bc9bfbba501630912814d56e5f",
    "balances": [
      {
        "amount": 77000000,
        "currency": "LBR"
      }
    ],
    "delegated_key_rotation_capability": false,
    "delegated_withdrawal_capability": false,
    "received_events_key": "0000000000000000f16df8bc9bfbba501630912814d56e5f",
    "sent_events_key": "0100000000000000f16df8bc9bfbba501630912814d56e5f",
    "sequence_number": 0
  }
}
```

## Expected Behavior

The specification should be aligned with the actual behavior.

**Comment by thefallentree** [comment]
@ssinghaldev can you fix this ?

**Comment by ssinghaldev** [comment]
yes I will take it!

**Comment by thefallentree** [comment]
as discussed, should add `get_currencies` calls as well

**Comment by ssinghaldev** [comment]
Submitted PR: https://github.com/libra/libra/pull/4169

**Comment by ssinghaldev** [comment]
Fixed the issue with #4169. Closing it
